tests/fixtures/custom_task_interface_transform_missing_values/custom.py

```diff
@@ -11,9 +11,50 @@
 
 class CustomTask(TransformerInterface):
     def fit(self, X, y, **kwargs):
-        # compute medians for all numeric features on training data, store them in a dictionary
+        """ This hook defines how DataRobot will train this task. Transformers will typically learn and/or
+        store information from training data and then apply it, e.g. learning the median for each column
+        and then transforming missing/NaN values to that median.
+        Note that when training a blueprint we apply this fit hook. After fit, we run the below transform hook on the
+        training data to pass to downstream tasks. In contrast, when scoring a blueprint on validation / holdout data
+        we only use the below transform hook.
+        The input parameters are passed by DataRobot based on project and blueprint configuration.
+        The output is the trained Custom Task instance, which allows a user to easily test, e.g.
+        task.fit(...).transform(...) or task.fit(...).save().transform(...)
+
+        Parameters
+        -------
+        X: pd.DataFrame
+            Training data that DataRobot passes when this task is being trained.
+        y: pd.Series
+            Project's target column (None is passed for unsupervised projects). This allows users to implement
+            transformers that rely on the target values, e.g. target encoding
+
+        Returns
+        -------
+        CustomTask
+            returns an object instance of class CustomTask that can be used in chained method calls
+        """
+
+        # Note that for transforms we typically save everything used in the transform hook below to self
```

Comment:
This comment sounds confusing. It should be something like `Any information derived from the training data (ie median values for each column) should be stored to self. Then, in the transform hook below, we use this information to transform any data that passes through this task.`